indy-vdr-proxy/src/app.rs

```diff
@@ -11,7 +12,7 @@ pub struct Config {
 }
 
 pub fn load_config() -> Result<Config, String> {
-    let matches = App::new("indy-vdr-proxy")
+    let mut app = App::new("indy-vdr-proxy")
```

Comment:
Check warning on line 15 in indy-vdr-proxy/src/app.rs

 GitHub Actions
/ Run Checks (windows-latest)

indy-vdr-proxy/src/app.rs#L15
variable does not need to be mutable